### PR TITLE
Update documentation for R2.10 (WIP)

### PR DESCRIPTION
**Pull request by inc1t3Ful**
** still need to update FeatureChangelist v2
** included are fixes to doc standardization as per (12142)

### CODE REVIEW

**Comment by arvindsree** [comment]
Seems to be having merge issues. reset and cherry-pick or rebase

File: website/docs/guides/release-compatibility.html.markdown [hunk 1]
```diff
@@ -50,6 +50,7 @@ Quick at-a-glance access to Aviatrix Terraform provider's release compatibility
 | 0.12              | 2.7                      | UserConnect-5.1              |
 | 0.12              | 2.8                      | UserConnect-5.1, 5.2         |
 | 0.12              | 2.9                      | UserConnect-5.2              |
+| 0.12              | 2.9.1                    | UserConnect-5.2.2122         |
```
Comment (arvindsree) [inline website/docs/guides/release-compatibility.html.markdown:53]: We should probably cherry-pick this onto the stable website branch

File: website/docs/guides/release-notes.html.markdown [hunk 2]
```diff
@@ -12,11 +12,21 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
...
+## 2.9.1
+### Notes:
+- Release date: **(28 Jan 2020)**
+- Supported Controller version: **UserConnect-5.2.2122**
+- Supported Terraform version: **v0.12.x**
+
+### Bug Fixes:
+1. Fixed issue where JSON Decode ``get_site2cloud_conn_detail`` fails for **aviatrix_site2cloud** and **aviatrix_vgw_conn**
+
+
```
Comment (arvindsree) [inline website/docs/guides/release-notes.html.markdown:29]: same

**Review by arvindsree — APPROVED** [review]

File: website/docs/guides/release-compatibility.html.markdown (hunk 1 above)
Comment (zhiqiangjin) [reply website/docs/guides/release-compatibility.html.markdown]: I will do it.

**Pull request merged**